Extensions/xkit_main.js

```diff
@@ -166,7 +166,7 @@ XKit.extensions.xkit_main = new Object({
 		}
 
 		try {
-			eval(xkit_main.script);
+			eval(xkit_main.script + "\n//# sourceURL=xkit/"+extension_id+".js");
```

Comment:
Nope it's just a misleading name. They're definitely evaluating extensions.
(I'm on mobile so can't double check, but I've been using this heavily for
debugging for about a five days, so I'm very sure I got it correct)
On Mon, Aug 17, 2015 at 5:49 PM James Hobin <notifications@github.com>
wrote:

> In Extensions/xkit_main.js
> <https://github.com/new-xkit/XKit/pull/299#discussion_r37246454>:
>
> > @@ -166,7 +166,7 @@ XKit.extensions.xkit_main = new Object({
> >  		}
> >
> >  		try {
> > -			eval(xkit_main.script);
> > +			eval(xkit_main.script + "\n//# sourceURL=xkit/"+extension_id+".js");
>
> Also same as above. It's worth looking at whether these reevaluations can
> be skipped in a future issue (I removed the first one of these at some
> point with no ill effects, but put it back in because paranoia)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/299/files#r37246454>.
>